packages/frontend/core/src/components/doc-properties/manager/index.tsx

```diff
@@ -98,44 +99,50 @@ const PropertyItem = ({
   );
 
   return (
-    <div
-      className={styles.itemContainer}
-      ref={elem => {
-        dropTargetRef.current = elem;
-        dragRef.current = elem;
-      }}
-      onClick={handleClick}
-      data-testid="doc-property-manager-item"
+    <Tooltip
+      content={t.t(
+        `com.affine.page-properties.property.${propertyInfo.type}.tooltips`
```

Comment:
add this to `packages/frontend/core/src/components/doc-properties/types/constant.tsx` ?